### PR TITLE
Dataflow Runner 0.3.0

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

**Comment by BenFradet** [comment]
@alexanderdean as promised :+1:

**Comment by alexanderdean** [comment]
Final edits from me. (Please leave the avatar as is - it will be updated when we have the photos done.)

**Comment by BenFradet** [comment]
@alexanderdean reverted the lock paths to being relative since a key starting with / is not authorized in consul.

**Comment by alexanderdean** [comment]
You reverted the path for the *local* file example though? The Consul example already uses a "relative" path. If we are going to use a relative path for the local file example, fine but you need to clarify what it's relative *to*...

**Comment by BenFradet** [comment]
I reverted it to be consistent, I didn't want to have different lock names for local and distributed.

If that doesn't make a lot of sense, I'm happy to revert back.

**Comment by alexanderdean** [comment]
Okay good point, let's keep it consistent. Please update the blog post with how the relative path for a local lock file works, I think that is more helpful to explain than using an absolute path, which is the easy use case anyway...

File: _posts/2017-05-30-dataflow-runner-0.3.0-released.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+{% highlight bash %}
+WARN[0000] lock already held at path/to/lock
+{% endhighlight %}
+
+You can set the lock name as appropriate to setup locks across different playbooks, job names and/or
+cluster IDs.
+
+In a local context, the lock will be materialized by a file at the specified path which can be
+relative or absolute. In case of a relative path, it will be relative to your current working
+directory.
```
Comment (BenFradet) [inline _posts/2017-05-30-dataflow-runner-0.3.0-released.md]: @alexanderdean added the explanation

**Comment by alexanderdean** [comment]
Explanation looks good, Iglu Central is next up for me...

**Pull request merged**